Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.yml

```diff
@@ -0,0 +1,218 @@
+          description: 'Connection name'
+          type: String
+        - contextPath: OctoxLabs.Connections.results.status
+          description: 'Connection status'
+          type: Boolean
+        - contextPath: OctoxLabs.Connections.results.description
+          description: 'Connection description'
+          type: String
+        - contextPath: OctoxLabs.Connections.results.enabled
+          description: 'Connection is enabled?'
+          type: Boolean
+    - name: octoxlabs-get-discoveries
+      deprecated: false
+      description: 'Fetch octoxlabs discoveries'
+      execution: false
+      arguments:
+        - default: true
+          description: 'Page number'
+          isArray: false
+          name: page
+          required: false
+          secret: false
+      outputs:
+        - contextPath: OctoxLabs.Discoveries.count
+          description: 'Total discovery count'
+          type: Number
+        - contextPath: OctoxLabs.Discoveries.results.id
+          description: 'Discovery id'
+          type: Number
+        - contextPath: OctoxLabs.Discoveries.results.start_time
+          description: 'Discovery start time'
+          type: String
+        - contextPath: OctoxLabs.Discoveries.results.end_time
+          description: 'Discovery end time'
+          type: String
+        - contextPath: OctoxLabs.Discoveries.results.status
+          description: 'Discovery status'
+          type: Number
+        - contextPath: OctoxLabs.Discoveries.results.hr_status
+          description: 'Discovery human readable status'
+          type: String
+        - contextPath: OctoxLabs.Discoveries.results.progress
+          description: 'Discovery progress'
+          type: Number
+    - name: octoxlabs-get-last-discovery
+      deprecated: false
+      description: 'Get last success discovery'
+      execution: false
+      outputs:
+        - contextPath: OctoxLabs.Discovery.id
+          description: 'Last discovery id'
+          type: Number
+        - contextPath: OctoxLabs.Discovery.start_time
+          description: 'Last discovery start time'
+          type: String
+        - contextPath: OctoxLabs.Discovery.end_time
+          description: 'Last discovery end time'
+          type: String
+        - contextPath: OctoxLabs.Discovery.status
+          description: 'Last discovery status'
+          type: Number
+        - contextPath: OctoxLabs.Discovery.hr_status
+          description: 'Last discovery human readable status'
+          type: String
+        - contextPath: OctoxLabs.Discovery.progress
+          description: 'Last discovery progress'
+          type: Number
+    - name: octoxlabs-search-devices
+      deprecated: false
+      description: 'Search in your devices'
+      execution: false
+      arguments:
+        - default: null
+          description: 'Query'
+          isArray: false
+          name: query
+          required: false
+          secret: false
+        - default: null
```

Comment:
As I said in the demo, let's make this argument clearer.
Please add a link to a guide (if such guide exists) about how to construct a query for your platform (as this is an advanced query type).

Reply:
We don't have a guide for query generate yet. I will add link when guide is ready. Also, we have a wizard to create query easily.